examples/abitest/client/abitest.cc

```diff
@@ -177,5 +205,10 @@ int main(int argc, char** argv) {
     storage_thread->join();
   }
 
+  if (grpc_test_thread != nullptr) {
+    grpc_test_server->Shutdown(std::chrono::system_clock::now() + std::chrono::milliseconds(100));
```

Comment:
Not sure we care enough, but AFAICT `system_clock` is not monotonic, so in some cases it may jump backwards or forwards, which would cause some odd behaviour. Does `steady_clock` work as a replacement here? It's unclear to me exactly what `Shutdown` expects, the template arg is very vague.

Reply:
I don't think it's worth worrying about the chances of a non-monotonic clock jump happening at the moment a test closes down.

Reply:
Probably not, but now we have a bit of code that will be copy-pasted somewhere else and cause some bug at 2 am when daylight saving time ends :) I was just curious if it is sufficient to use a different clock and it perhaps everything still works.